src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,45 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int MAX_LENGTH_OF_STORAGE = 10;
+    private final Object[] keysStorage;
+    private final Object[] valuesStorage;
+    private int size;
+
+    public StorageImpl() {
+        keysStorage = new Object[MAX_LENGTH_OF_STORAGE];
```

Comment:
To initialize generic array you need to perfom type conversion here:
```suggestion
        keysStorage = (K[]) new Object[MAX_LENGTH_OF_STORAGE];
```